.github/workflows/lambda.yaml

```
@@ -0,0 +1,62 @@
+name: Build and deploy lambda function
+
+on:
```

I think we should also upload when a new version is tagged too, since we may forget to manually trigger the workflow:

```suggestion
on:
  push:
    tags:
    - 'v*'
```